### DISCUSSION TITLE
Thoughts on GitHub no longer accepting requests to release dormant usernames

### DISCUSSION THREAD

**Discussion by kimmy-wang** [community/community#20432] [Other Feature Feedback, Questions, & Ideas]
@ashtom I personally think that it will cause a waste of resources, and releasing some dormant usernames can stimulate vitality. It is recommended that GitHub re-accept requests for releasing some dormant usernames, and review the use of the request.

**Comment by jfullstackdev** [comment]
that was really a very long thread in the old GitHub Community Forum

as for me, there will always be new unique names we can use, actually I've changed mine three times
